openassessment/xblock/submission_mixin.py

```diff
@@ -702,6 +706,43 @@ def render_submission(self, data, suffix=''):  # pylint: disable=unused-argument
         path, context = self.submission_path_and_context()
         return self.render_assessment(path, context_dict=context)
 
+    def get_team_submission_context(self, context):
+        """
+        Populate the passed context object with team info, including a set of students on
+        the team with submissions to the current item from another team, under the key
+        `team_members_with_external_submissions`.
+
+        Args:
+            context (dict): render context to add team submission context into
+        Returns
+            (dict): context arg with additional team-related fields
+        """
+
+        from submissions import team_api
+        team_info = self.get_team_info()
+        if team_info:
+            context.update(team_info)
+
+            submitter_anonymous_user_id = self.xmodule_runtime.anonymous_student_id
+            student_item_dict = self.get_student_item_dict(
+                anonymous_user_id=submitter_anonymous_user_id
+            )
+            external_submissions = (
+                team_api.get_teammates_with_submissions_from_other_teams(
+                    self.course_id,
+                    student_item_dict["item_id"],
+                    team_info["team_id"],
+                    self.get_anonymous_user_ids_for_team()
+                )
+            )
+
+            team_usernames = list_to_conversational_format([
+                self.get_username(submission['student_id']) for submission in external_submissions
+            ])
+            context["team_members_with_external_submissions"] = team_usernames
+            return context
```

Comment:
no point in returning anything, no?

Reply:
was using it for the test, but you aren't wrong.  updating.